front/app/utils/i18n.ts

```diff
@@ -84,3 +84,25 @@ export function getLocalized(
 
   return '';
 }
+
+export function getLocalizedWithFallback(
```

Comment:
Added this wrapper around `getLocalized` to add support for a `fallback` string, but only for regular multilocs (i.e. not for graphql-multiloc, whatever those are).

Reply:
That's super useful!